wiki/Installation-Instructions.md

```diff
@@ -1,59 +1,36 @@
+The mesh adaptation modules are dependent on a custom setup for Firedrake with PETSc from either a [local installation](#local-firedrake-installation) or a [pre-built Docker image](#installing-firedrake-via-docker-image).
 Once Firedrake is installed (or if you already have a working installation), see the section on [installing Animate, Goalie, or Movement](#installing-animate-goalie-or-movement).
 
-## Installing Firedrake with custom PETSc
+## Local Firedrake installation
 
-Firedrake and PETSc are required by Animate, Goalie, and Movement. Choose from the following installation approaches for these packages.
+As described on the [Firedrake installation page](https://www.firedrakeproject.org/install.html#), a native installation of Firedrake is accomplished in 3 steps:
+1. Installing systems dependencies
+2. Installing PETSc
+3. Installing Firedrake itself.
 
-### Default approach
-
-The first approach uses the default MPI packages found in the path.
-Run the following code to download and install Firedrake and PETSc using the associated Makefile and bespoke PETSc options:
-```
-curl -O https://raw.githubusercontent.com/mesh-adaptation/mesh-adaptation-docs/main/install/Makefile
-curl -O https://raw.githubusercontent.com/mesh-adaptation/mesh-adaptation-docs/main/install/petsc_configure_options.txt
-make install
-```
-
-The `make install` command accepts several optional arguments:
-* `FIREDRAKE_ENV` for setting the Firedrake virtual environment name. Defaults to `firedrake-mmmyy`, where `mmm` is the first three letters of the current month and `yy` are the last two digits of the current year.
-* `BUILD_DIR` for setting the build directory. Defaults to the current directory.
-* `PETSC_CONFIGURE_FILE` for setting the PETSc configure options. Defaults to `petsc_configure_options.txt`.
-These options may be combined, e.g.:
-```
-make install FIREDRAKE_ENV=firedrake-dev BUILD_DIR=/scratch PETSC_CONFIGURE_FILE=my_configure_file.txt
-```
-
-### Custom MPI approach
-
-The second approach allows customisation of the MPI packages by passing arguments to the associated Makefile. First, download the Makefile and bespoke PETSc options using `curl`:
-```
-curl -O https://raw.githubusercontent.com/mesh-adaptation/mesh-adaptation-docs/main/install/Makefile
-curl -O https://raw.githubusercontent.com/mesh-adaptation/mesh-adaptation-docs/main/install/petsc_configure_options.txt
-```
-When running `make install`, pass arguments for `MPICC`, `MPICXX`, `MPIF90`, and `MPIEXEC`:
+To use the mesh adaptation modules, we can install system dependencies (step 1) and Firedrake (step 3) exactly as described, but we must [customise the PETSc installation](https://www.firedrakeproject.org/install.html#id29) (step 2) to install additional packages: chaco, eigen, parmetis, mmg, and parmmg. We do that simply by passing the `--download-<PACKAGE>` flags when running PETSc `configure`:
```

Comment:
I don't know that we actually need chaco? I have a feeling this is a legacy dependency.